Example/Example/ContentView.swift

```diff
@@ -5,22 +5,32 @@
 //  Created by David Bui on 20/03/2023.
 //
 
+import NimbleLoyalty
 import SwiftUI
+import UIKit
+import WebKit
 
 struct ContentView: View {
     var body: some View {
         VStack {
             Image(systemName: "globe")
                 .imageScale(.large)
                 .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            Text("Hello Nimble!")
         }
         .padding()
-    }
-}
-
-struct ContentView_Previews: PreviewProvider {
-    static var previews: some View {
-        ContentView()
+        .onAppear {
+            DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+                NimbleLoyalty.shared.setClientId("MRC6bwFASOm0sJYPYwVEt2KD51bkAtdgS7ltqeDYUf4")
```

Comment:
Please use `arkana` to setup all secret keys (including client id) for the example app.